bitwarden_license/src/Commercial.Core/SecretsManager/Commands/Porting/ImportCommand.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Bit.Core.SecretsManager.Commands.Porting;
+using Bit.Core.SecretsManager.Commands.Porting.Interfaces;
+using Bit.Core.SecretsManager.Entities;
+using Bit.Core.SecretsManager.Repositories;
+
+namespace Bit.Commercial.Core.SecretsManager.Commands.Porting;
+
+public class ImportCommand : IImportCommand
+{
+    private readonly IProjectRepository _projectRepository;
+    private readonly ISecretRepository _secretRepository;
+
+    public ImportCommand(IProjectRepository projectRepository, ISecretRepository secretRepository)
+    {
+        _projectRepository = projectRepository;
+        _secretRepository = secretRepository;
+    }
+
+    public async Task<SMImport> ImportAsync(Guid organizationId, SMImport import)
+    {
+        var importedProjects = new List<Project>();
+        var importedSecrets = new List<Secret>();
+
+        try
+        {
+            import = AssignNewIds(import);
+
+            if (import.Projects != null && import.Projects.Any())
```

Comment:
This check can never be false since we set the values in AssignNewIds.

Reply:
Removed the `import.Projects != null` check